src/font_loader.h

```diff
@@ -15,16 +15,30 @@
 #include "json.h"
 #include "path_info.h"
 
+// Represents the possible hinting options for a given font.
+// The names and values are taken from ImGuiFreeTypeBuilderFlags.
+enum class FontHint {
+    Default = 0,
+    NoAuto = 1 << 1,
+    Auto = 1 << 2,
+    Light = 1 << 3,
+    Bitmap = 1 << 9,
+};
+
+using font_config = std::pair<std::string, FontHint>;
```

Comment:
Is there a compelling reason to have our own `FontHint` enum? We could use the values from `ImGuiFreeTypeBuilderFlags` instead.

Reply:
`ImGuiFreeTypeBuilderFlags` doesn't have a value for default, so I would need to use a bare integer, a `std::optional<ImGuiFreeTypeBuilderFlags>`, or modify imgui. It also allows custom configuration options more easily.